packages/synchro-charts/src/components/sc-dial/sc-dial-base/sc-dial-svg.tsx

```diff
@@ -0,0 +1,118 @@
+import { Component, h, Prop } from '@stencil/core';
+import { NO_VALUE_PRESENT } from '../../../constants';
+import { DataPoint, DataStream, DialSizeConfig } from '../../../utils/dataTypes';
+import { round } from '../../../utils/number';
+import { getIcons } from '../../charts/common/annotations/iconUtils';
+import { Threshold } from '../../charts/common/types';
+import { sizeConfigurations } from './util';
+
+@Component({
+  tag: 'sc-dial-svg',
+  shadow: false,
+})
+export class scDialSVG {
```

Comment:
should be capitalized

Reply:
Done --- Change ‘scDialSVG’ to ‘ScDialSvg’ .